docs/CommandLine.md

```diff
@@ -0,0 +1,150 @@
+# Command Line
+
+Rokul Runnings has a CLI to execute all commands in the Rokul Runnings class via the command line.
+
+## Setup
+
+### Install
+
+Rokul Runnings CLI can be set up to run either globally or at a project level. In order to use either, they must be installed:
+
+```
+// global
+npm install -g @willowtreeapps/rokul-runnings
+```
+
+```
+// project level
+npm install @willowtreeapps/rokul-runnings
+```
+
+### Defining Config values
+
+Rokul Runnings CLI creates an instance of the Rokul Runnings class based on defaults specified in a config JSON file. The structure of that file looks like:
+
+```json
+{
```

Comment:
update these comments because VSCode led you astray that there wouldn't be any errors aka you forgot that JSON can't have comments.

Reply:
Fixed in 4ae4d34